### PR TITLE
Fix description linking in compat tables

### PR DESCRIPTION
**Pull request by Elchi3**
This should fix https://github.com/mdn/kumascript/issues/558

### CODE REVIEW

File: macros/CompatBeta.ejs [hunk 1]
```diff
@@ -105,9 +105,10 @@ function writeFeatureName(row, feature) {
   } else {
     desc += `<code>${Object.keys(row)[0]}</code>`;
   }
-  // Add relative MDN url, but don't add it for "Basic support"
-  if (feature.mdn_url && label != identifier) {
-    desc = `<a href="${feature.mdn_url.replace('https://developer.mozilla.org', '')}">${desc}</a>`;
+  // Add relative MDN url, but don't link to the current page
+  let href = feature.mdn_url.replace('https://developer.mozilla.org', '');
+  if (href && href != '/docs/' + env.slug) {
```
Comment (schalkneethling) [inline macros/CompatBeta.ejs]: Is the use of a single equals sign here intentional?

Comment (Elchi3) [reply macros/CompatBeta.ejs]: No, good catch!

**Comment by schalkneethling** [comment]
@Elchi3 There also seem to be a bunch of test failures. I restarted the build job thinking it might be an intermittent thing, but it has failed again :-/

**Comment by Elchi3** [comment]
The test failure was amazingly right and helped me to write the correct thing here. Woah!

**Pull request merged**